contracts/CrossChain.sol

```diff
@@ -35,6 +35,9 @@ contract CrossChain is System, ICrossChain, IParamSubscriber{
   mapping(uint8 => uint64) public channelReceiveSequenceMap;
   mapping(uint8 => bool) public isRelayRewardFromSystemReward;
 
+  // to prevent forged header
```

Comment:
`to prevent forged header` is not ture,  `to prevent the utilization  of ancient block header`

Reply:
Fixed